uwblueprint/jump-math
# feat: create classroom using mutation from create classroom modal

## Notion ticket link
[Create mutation to Create Classroom](https://www.notion.so/uwblueprintexecs/Create-mutation-to-Create-Classroom-16dd181c8ab8492c9ec3ac8f26f2141c?pvs=4)



## Implementation description
* Created `frontend/src/APIClients/mutations/ClassMutations.ts` to call `createClass` mutation from frontend
* Made changes to classroom modal and related front end components to call `createClass` mutation



## Steps to test
1. Test front end 
2. Check MongoDB atlas when saving classes to make sure all the fields are filled in properly


## Checklist
